### PR TITLE
Made AvgCycleScreen

### LINKED ISSUE #49: Add new design to AvgCycleScreen
Change the current AvgCycleScreen page to this 

image

Check wireframe for files.

### PR DESCRIPTION
**Pull request by Rits1272**

### CODE REVIEW

**Comment by Rits1272** [comment]
Closes #49. Please review @sarthology

**Comment by sarthology** [comment]
There is a bug in it. 
image

File: src/screens/ProfileScreen/AvgCycleScreen.js [hunk 1]
```diff
@@ -1,86 +1,120 @@
...
+          renderItem={({ item }) => (
+            <TouchableOpacity onPress={() => this.setState({ cycle: item })}>
+              <Text style={[this.state.cycle === item ? styles.selected : styles.number]}>
+                {item}
+              </Text>
+            </TouchableOpacity>
+          )}
         />
-
-        <Picker
```
Comment (sarthology) [inline src/screens/ProfileScreen/AvgCycleScreen.js:23]: We have to use picker only.

**Review by sarthology — CHANGES_REQUESTED** [review]

**Comment by Rits1272** [comment]
Will it suffix @sarthology??

**Comment by sarthology** [comment]
Use this 
https://github.com/rdhox/react-native-smooth-picker

**Comment by Rits1272** [comment]
Please review the changes @sarthology

**Comment by sarthology** [comment]
Works for now.

**Pull request merged**